theme_enark/views/image_library.xml

```diff
@@ -188,7 +188,7 @@
 <record id="s_accordion_image_default_image" model="theme.ir.attachment">
     <field name="key">website.s_accordion_image_default_image</field>
     <field name="name">website.s_accordion_image_default_image</field>
-    <field name="url">/theme_enark/static/src/img/snippets/library_image_10.jpg</field>
+    <field name="url">/theme_enark/static/src/img/snippets/s_shape_image_default_image.jpg</field>
 </record>
```

Comment:
Just making sure this is intended: changing the accordion default image - rather than defining an image for this snippet ?